CMakeLists.txt

```diff
@@ -685,6 +685,8 @@ if(NRN_ENABLE_PYTHON)
   message(STATUS "  DYNAMIC     | ${NRN_ENABLE_PYTHON_DYNAMIC}")
   if(PYTHON_VERSION_MAJOR EQUAL 2)
     set(PY2_WARNING TRUE)
+  elseif(PYTHON_VERSION_MINOR EQUAL 5)
+    set(PY35_WARNING TRUE)
```

Comment:
So `4.5` is not considered? 😉  But if the idea is to remove this right after 8.0 then it's ok for me.

Reply:
yes, didn;t want to spend much time on it. It was off the bat, no testing :) but I confirm it works from azure build logs